libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -98,7 +98,7 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Param: QNH_PRESSURE
     // @DisplayName: QNH pressure
     // @Description: This sets the QNH pressure in millibars to be used for pressure altitude in the altitude limit. A value of zero disables the altitude limit.
```

Comment:
The description needs to change as well.
BTW are the units equal? there is no need to change the code ?

Reply:
they are equal

Reply:
Yes, they are equal.  But @amilcarlucas is pointing out that if we are going to canonicalise on a single unit it should be done right.

Reply:
@peterbarker what do you mean ? I didn't find another parameter using milibar or bar